pkg/logicalbackup/logicalbackup.go

```diff
@@ -38,6 +39,16 @@ const (
 	cDelete
 )
 
+type NameChangeHistory struct {
+	Name message.NamespacedName
+	Lsn  dbutils.Lsn
+}
+
+type OidToName struct {
+	IsChanged bool
```

Comment:
that struct is used only 1 time in the `LogicalBackup` struct, I suggest moving those two struct fields directly to `LogicalBackup` struct

Reply:
Didn't want to add multiple fields to the logical backups, as those fields are closely tied to each other, therefore the struct.  With c5eb38a the struct variable is used in many more places.